index.html

```diff
@@ -149,6 +149,37 @@ <h4>Skills</h4>
 
     </section>
 
+    <section class="form">
+        <img src="images/logo.svg" alt="">
+        <h1>Get started</h1>
+        <p>Lorem ipsum dolor sit amet, consectetur adipiscing elit,
+             sed do eiusmod tempor incididunt ut labore et dolore magna.</p>
+
+        <form class="form-start" action="https://formspree.io/f/mwkjeepg" method="POST">
+            <input class="name" type="text" name="name" id="name" maxlength="30" placeholder="Write your name" required >
+            <input class="email" type="email" name="email" id="email" placeholder="Write your email" required >
+            <input class="message" type="text" name="message" id="message" maxlength="500" placeholder="Enter your message here" required >
```

Comment:
As requested in the project requirements:
![image](https://user-images.githubusercontent.com/42799579/207972331-c409319b-9cf2-48f8-a681-747475e015d5.png)

Kindly notice that your `message` field should be a `textarea` element 😉. [Learn more about it here.](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/textarea)